tests/pipelines/eks/awscli-eks-cl2-load.yaml

```diff
@@ -11,8 +11,10 @@ spec:
   - name: cluster-name
   - name: endpoint
   - name: servicerole
+    default: ""
   - name: desired-nodes
   - name: host-cluster-node-role-arn
+    default: ""
```

Comment:
same with this, you can remove this at pipeline level as we are anyway using this is task level default.

Reply:
same here